test/args_test.dart

```diff
@@ -184,6 +184,40 @@ void main() {
     });
   });
 
+  group('ArgParser.findByNameOrAlias', () {
```

Comment:
Can we add a test that validates aliases are used in `ArgParser.parse()`?

Reply:
done